WooCommerce/WooCommerceTests/ViewModels/CardPresentPayments/PaymentReceiptEmailParameterDeterminerTests.swift

```diff
@@ -0,0 +1,129 @@
+import XCTest
+import TestKit
+@testable import WooCommerce
+@testable import Yosemite
+
+final class PaymentReceiptEmailParameterDeterminerTests: XCTestCase {
+    private var stores: MockStoresManager!
+
+    override func setUp() {
+        super.setUp()
+
+        stores = MockStoresManager(sessionManager: SessionManager.makeForTesting())
+        stores.whenReceivingAction(ofType: SystemStatusAction.self) { action in
+            guard case let .synchronizeSystemPlugins(_, onCompletion) = action else {
+                return
+            }
+
+            onCompletion(.success(()))
+        }
+    }
+
+    override func tearDown() {
+        stores = nil
+    }
+
+    func test_when_only_WCPay_is_active_and_version_is_higher_than_minimum_that_sends_email_then_returns_nil() {
+        let order = Order.fake()
```

Comment:
nit: we usually add `Given/When/Then` comments to test cases, so that it's easier to read the code for each part 🙂 (similarly for other test cases in this file)

```suggestion
        // Given
        let order = Order.fake()
```

Reply:
Good idea! added